telegram/bot.py

```diff
@@ -510,6 +515,9 @@ def send_message(
         if entities:
             data['entities'] = [me.to_dict() for me in entities]
 
+        if protect_content:
+            data['protect_content'] = protect_content
+
```

Comment:
I'd vote to just make this a parameter of `self._message` - I didn't check, but (almost) all methods that got this parameter now call `self._message` anyway and this saves us the `if protected_content:` :)